config/locales/devise.en.yml

```diff
@@ -8,10 +8,10 @@ en:
     failure:
       already_authenticated: You are already signed in.
       inactive: Your account is not activated yet.
-      invalid: Invalid %{authentication_keys} or password.
+      invalid: Unable to login.
       last_attempt: You have one more attempt before your account is locked.
-      locked: Your account is locked.
-      not_found_in_database: Invalid %{authentication_keys} or password.
+      locked: Unable to login.
+      not_found_in_database: Unable to login.
       timeout: Your session expired. Please sign in again to continue.
       unauthenticated: You need to sign in or sign up before continuing.
       unconfirmed: You have to confirm your email address before continuing.
```

Comment:
I think we should apply our approach to the `unconfirmed` scenario as well. @JennieOcken